web/js/jquery.websocket.js

```diff
@@ -35,7 +35,12 @@
                     .bind('message', function(e) {
                         var m = JSON.parse(e.originalEvent.data);
                         var h = settings.events[m.type];
-                        if (h) h.call(this, m);
```

Comment:
Perhaps. I'm not very familiar with what's going on here, it took me a while to even find this point in the code. But I believe I'm now catching unknown types in our code, I'll try adding a handler for settings.event["undefined"] and see if that works.